src/containers/Header/HelpButton.js

```diff
@@ -1,22 +1,58 @@
-import React from 'react';
-import { connect } from 'react-redux';
+import React, { useState } from 'react';
+import { useTranslation } from "react-i18next";
+import ReactMarkdown from 'react-markdown';
+import rehypeRaw from 'rehype-raw';
+import { useSelector } from 'react-redux';
 
-import { NavLink } from 'reactstrap';
+import { Modal, NavLink, Card, CardHeader, CardBody, Button } from 'reactstrap';
 
-const HelpButton = ({ url, icon = "cil-info", target="_blank" }) => {
-    if (!url) return null;
+
+export default function HelpButton() {
+    const { t } = useTranslation();
+
+    const [modal, setModal] = useState(false);
+
+    const content = useSelector(state => state?.helpButton.content);
+    if ((content == undefined) || (content == "")) return null;
+
+    const toggle = () => setModal(!modal);
 
     return (
-        <NavLink 
-            style={{ fontSize: "1.2rem", marginRight: "1rem" }}
-            href={url}
-            target={target}
-        >
-            <i className={icon}></i>
-        </NavLink>
+        <>
+            <NavLink
+                className="help-button"
+                onClick={toggle}
+                title={t("Show help info")}
+                href="#"
+            >
+                <i>?</i>
+            </NavLink>
+
+            <Modal isOpen={modal} toggle={toggle} className="center help-modal-card">
+                <Card>
+                    <CardHeader className="border-bottom">
+                        <div className="card-header-title">
+                            <i className="cil-info pr-2" />
+                            {t("HelpButton|Help")}
```

Comment:
@Aringoaway Add those locales to `public` of demo app as well. Czech translation will be `Nápověda`